docs/pages/operations-guide.md

```diff
@@ -583,29 +624,29 @@ Given the above example, let's build an `elastic-prod-options.json` which custom
 }
 ```
 
-Given the above `elastic-prod-options.json`, we can install a service instance that uses it as follows:
+Now that we have `elastic-prod-options.json`, we can install a service instance that uses it as follows:
 
 ```bash
 $ dcos package install --options=elastic-prod-options.json elastic
 ```
 
-Once we know the configuration is good, we will then add it to our source control for tracking.
+Once we know the configuration is good, we can add it to our source control for tracking.
 
 ## Updating service configuration
 
-Early on, we described how a configuration update is handled. Now we will quickly show the steps to perform such an update.
+Above, we described how a configuration update is handled. Now we will quickly show the steps to perform such an update.
 
 Configuration updates are performed by updating the process environment of the Scheduler. The Scheduler runs as a Marathon application, so we can perform the change there.
 
-First, we can go to `https://yourcluster.com/marathon` and find the Scheduler App in Marathon. In this case we'll use `dse`:
+First, we can go to `<dcos-url>/marathon` and find the Scheduler App in Marathon. In this case we'll use `dse`:
 
 [<img src="img/ops-guide-marathon-app-list.png" alt="list of Marathon apps" width="400"/>](img/ops-guide-marathon-app-list.png)
 
-We click on the `dse` app and then the `Configuration` tab. In here we see an `Edit` button tucked away off to the right of the screen:
+We click on the `dse` app and then the `Configuration` tab. In here we see an `Edit` button tucked away off to the right of the screen: <!-- we can do this from the DC/OS UI, right? -->
```

Comment:
Yes, however:
- I've personally had that UI fail on me even when I was using it correctly -- figured it'd be best to avoid stacking DC/OS problems on top of whatever problems the operator may be having with their service.
- The UI has changed how this works in 1.8 vs 1.9 (now the edit option is obscured behind a menu) so this section would need to document both UIs.
- The 1.9 UI in particular has a bug where if you mouseover a listed environment variable, an unusable 'edit' option appears. I imagined most users would mess up and hit that instead of the edit option that actually works, particularly given the aforementioned obscuring of that edit option.